### DISCUSSION TITLE
大家會留這學期用完的書嗎?

### DISCUSSION THREAD

**Discussion by yuchien-lin** [1101-GS4719-C/Python-Community-GS4719-C#1879] [General]
有些只上了一學期的課，之後用不到的書大家會選擇拿去賣二手的嗎?還是會留下來當作紀念?又或是傳給下一屆?我是會選擇留下來當作紀念的，以後可以看一下大一時到底學過了什麼，或是之後可能考研究所時會用到，也可以拿來參考。

**Comment by RavenJiaSin** [comment]
我覺得如果有平板的話，電子書是一個不錯的選擇，管理方便，又不用帶著厚重的書跑來跑去。另外，如果你忽略智慧財產權與道德相關問題，還可以省掉買書的錢錢(邪惡的笑容)。

**Comment by vchien03** [comment]
我自從有了平板之後，就幾乎沒有用紙本書了，畢竟電子書不像紙本書需要占體積跟重量，而且很好整理，存在雲端上還可以隨時閱讀。
不過如果是紙本書的話，我喜歡留下來，偶爾翻一翻自己做的筆記，可以懷舊一下~

**Comment by thz0000** [comment]
我到現在除了大一英文沒買過半本原文書XD(但英文買了也沒在用，浪費我的錢錢，幸好後來有賣掉)
其他書都是直屬給我的，沒給的就是看電子書，但我大二才買平板，相見恨晚阿
所以沒意外的話也會把紙本書傳給直屬~

**Comment by chuanmin0317** [comment]
基本上現在如果有電子書的話，我就直接下載下來在平板上看了，畢竟大多數的原文書都比平板重很多，而且電子版的原文書可以直接搜尋關鍵字，在讀的時候方便了很多

**Comment by TobywithDino** [comment]
大一的時候還沒買平板，當時雖然都有看到同學在買書或是學長姐在送書，但是自己的經驗是覺得資工系大一的話除了英文國文完全不用買書XD
大二有了平板之後，電子書甚麼的就很方便，也不會有太笨重或是忘記帶書的問題，大推平板~

**Comment by lyykey** [comment]
有時候覺得買書沒有什麼用。但以我的經驗，之前在做數學或電路學時，其實有紙本的書會給我比較多的學習感覺，所以後來有買幾本紙本書，不過缺點就是要花很多錢。
到現在還有幾本書放在我家，更本也不會打開看，要賣好像也只能賣50%以下，我好像完蛋了。

**Comment by YifenLin** [comment]
我自己目前是使用電子書，之前買過的書都留給直屬了。現在直屬再用的書也是電子化。真心覺得有平板很方便，個人覺得是大學生涯中最值得的投資之一。現在上補習班，教材沒有提供電子版反而覺得不方便。

**Comment by 110101501** [comment]
倘若是實體書的話，據過往經驗是不會留而是直接送給直屬，一代傳一代。
不過自從使用平板作為閱讀的媒介後，就很少再度購買實體書了。

**Comment by YANRONG0123** [comment]
我之後可能會考慮使用平板，畢竟感覺裝備齊全，才能和其他人站在同一條起跑線上，也不用背著一堆用完就丟的書跑來跑去，每一學期的書，我其實很想要找個時間通通賣掉，畢竟很多書只有介紹性質，解題攻略還是要另外購買。

**Comment by YuanWang08** [comment]
我高中時期的書一本都沒有丟，包含補習班的講義也是，放滿了一整個書架。國中的書是因為放不下了才回收掉。對我而言，把書留著有點像是一種在保留回憶的事情，雖然那些書長大至今根本沒有回去翻過，但看著那些書在那邊，就好像有一種我以前很認真的感覺。所以我不喜歡丟書，有點像是想要留下一點我以前努力過的證明。

**Comment by Eileen050824** [comment]
我自己比較喜歡紙本書，覺得作筆記比較方便，不過上大學後發現原文書都很重(每次都會割書)，價格也不菲(常買較新二手書)，就有些非本科的書用電子版的。書的話我是分學習階段丟，高中丟國中的，大學丟高中的，會留一些重要的/補習班的下來，但每次清理的時候量都很驚人(一定會堆得比我還高XD)！

**Comment by tzurou** [comment]
自從買了平板之後我就很少買書了，原文書真的很貴又很重，現在有的書都是直屬傳好幾屆下來的但也沒用到，真心覺得可以買平板，也不是每堂課都會用課本，平板拿來看簡報、paper也很好用

**Comment by jylin34** [comment]
我也是通常下載課堂用書的電子檔用平板看，如果是重要的科目可能會特別買一本二手的實體原文書，有時候可以直接翻，不重要或我不喜歡的科目從一開始就不會買書。

**Comment by 710388** [comment]
會留，以免以後還需要看，而且我覺得看紙本還是比較舒服，不然每學期要看的螢幕時間實在太多了，如果看書也要看平板太累了。

**Comment by willychiuchiu** [comment]
以前高中的書上完基本上都會留著，覺得總有哪一天會有想要拿出來讀的時候，但事實證明是我多想了，全部都再也沒有碰過，於是後來就全部丟掉了。而上了大學以後買了平板，就沒有再買過實體書了，覺得平板真的是很偉大的發明，既可以永久保存書又不用佔空間，讚

**Comment by yxin214** [comment]
以前國高中的書都留很久，覺得想留下認真的痕跡，但大學的書用完就想趕快賣掉，因為一本書真的太貴了，而且怕不馬上賣掉等到有新版本出版時就沒人要跟我買了。

**Comment by wanka2030** [comment]
國高中的書有可能是因為價格的便宜所以痕容易就想要保存下來，因為裡面充滿手寫的筆記以及劃下的重點都成為了學生時期的回憶。但升到大學之後，會因為每本都是價格昂貴的外文書，都迫不及待將他以二手拋出、來換取現金生活，但還是對於用過的原文書仍然充滿回憶。

**Comment by thomasxshen** [comment]
不會，都給學弟了，我是一個很寵學弟的好直屬，希望我學弟也能把我花大錢買的書傳承下去

**Comment by EPHMR0524** [comment]
我因為用得都是電子書，所以基本上用完就會一直存在平板裡面，也不會刪掉，而且留著的好處是如果未來有事情需要裡面的知識，可以馬上翻閱而不月去和別人借書

**Comment by yyujimmy** [comment]
每個人對於舊書的處理方式可能都有所不同，取決於個人的偏好和情況。以下是一些可能的選擇：

賣二手書：有些人可能會選擇將只上了一學期的課程書賣給其他需要的學生，這樣可以回收一部分成本，也讓其他人有機會使用這些書籍。
留作紀念：像你所提到的，有些人可能會選擇留下來當作紀念，作為大學生活的一部分。這些書籍可能代表著他們的努力和學習成果，可以在日後回顧時帶來一些回憶。
傳給下一屆：有些人可能會將書籍傳給下一屆的學生，這樣可以幫助新生節省購書的成本，同時也是一種將知識傳承下去的方式。
每個選擇都有其自身的價值和意義，最重要的是找到最適合自己的方式，使得舊書能夠發揮最大的作用。

**Comment by KH1126** [comment]
目前都是用平板在電子書上記錄筆記，所以基本上不會買書。
但如果有買實體書的話，我會在下一學年起初把書賣掉，因為那時候需求最多，比較容易賣掉。

**Comment by YOGA012** [comment]
傳給下一屆有需要的學弟妹吧，我自己的書都是學長傳給我的，雖然我自己有平板也有電子書，但是學長的書裡面有豐富的前人筆記，讓我在課堂中可以較輕易掌握上課內容，下課後自己做筆記也比較輕鬆，所謂前人種樹後人乘涼，所以我會選擇繼續傳承。

**Comment by William08290829** [comment]
我覺得基本上就看我上面有沒有做很多筆記，如果有很多筆記我會把它拿來收藏，多年後拿來回憶會覺得蠻有價值的。但如果上面幾乎空白那就是拿去半價賣了，或是拿去送給有需要的學弟妹之類的。

**Comment by jeff110303549** [comment]
保留課本當作紀念是一個很好的主意！這樣可以回顧自己曾經學過的知識，也許在未來的學習或工作中會派上用場。此外，如果你覺得這些書對其他人也有價值，可以考慮捐贈給學校圖書館或是傳給下一屆學生，讓更多人受益。留下來的書籍也許還會成為你回憶大學時光的一部分，帶來溫馨的回憶和意義。
